### ISSUE TITLE
TP: add tooltips for routing methods on dashboard

### ISSUE THREAD

**Issue by mitchell852** [apache/trafficcontrol#4130]
## I'm submitting a ...

- [ ] bug report
- [ ] new feature / enhancement request
- [x] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [x] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:
![image](https://user-images.githubusercontent.com/251272/69178358-ab2ee100-0ac6-11ea-8eb1-218ace21f6c5.png)

It is unclear what each routing method means.

## Expected / new behavior:
Add a tooltip or something with a description of each value.

## Minimal reproduction of the problem with instructions:
Open the TP dashboard.

**Comment by ocket8888** [comment]
Instead of tooltips, maybe they could just have better names? E.g. I believe "Native" actually means "Used Coverage Zones".

**Comment by mitchell852** [comment]
> Instead of tooltips, maybe they could just have better names? E.g. I believe "Native" actually means "Used Coverage Zones".

agreed. not sure why `Native` was chosen to describe requests routed via the coverage zone file.

**Comment by ocket8888** [comment]
Well idk about originally, but "native" is the name of the enum member that represents that in the internal java code, and is the name of that metric in logs and api responses, so that's probably why portal uses it.
